### PR TITLE
Added details for Vim meetup

### PR DESCRIPTION
**Pull request by nightwarrior-xxx**
@darkHarry  @ryzokuken  Details of latest meetup added here .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,4 +16,4 @@ List of all the meetups of the OSDC Community along with the links to their resp
  2018-09-14 | [How to Open Source](https://bit.ly/2PWTA2Z)  |Addressing the know-hows of stepping into open-source.
  2018-09-28 | [Intro to Git & Github](blog-link-here)       |Introduction to version control using Git & Github.
  2019-01-23 | [Contributing to Node.js](blog-link-here)     |Getting started with contributing to Node.js.
- 
+ 2019-01-31 | [Intro to Vim](https://github.com/vaibhavk/presentations/blob/master/index.org)     |Introduction to Vim and its module.
```
Comment (darkHarry) [inline README.md]: The link must be pointing to the corresponding blog which as of now is not present.

Comment (nightwarrior-xxx) [reply README.md]: @darkHarry  Which blog ? There is only a index.org file on @vaibhavk presentations repo .

Comment (nightwarrior-xxx) [reply README.md]: @darkHarry  Someone has to the write blog . Can I ?

Comment (darkHarry) [reply README.md]: yes, you can.

**Comment by sidntrivedi012** [comment]
@nightwarrior-xxx I didn't see this PR and already added the details for the Vim meetup in mine. Can you please pull the changes and add the link to slides to it?

**Comment by nightwarrior-xxx** [comment]
@sidntrivedi012 at that time @vaibhavk told me that slides are in his [Github Repo](https://github.com/vaibhavk/presentations). Don't know if we should add it this way or not. Because slides were made using reveal js and org mode.

**Comment by sidntrivedi012** [comment]
Ig we can add it as of now like that or maybe put them on slides.com? What
do you say?

**Comment by ryzokuken** [comment]
@nightwarrior-xxx please resolve merge conflicts.

**Comment by nightwarrior-xxx** [comment]
@ryzokuken PTAL

**Pull request merged**